packages/js/src/analysis/TermDataCollector.js

```diff
@@ -102,25 +84,9 @@ TermDataCollector.prototype.getText = function() {
 };
 
 /**
- * Returns the meta description from the DOM.
+ * Returns the slug from the DOM.
  *
- * @returns {string} The meta.
- */
-TermDataCollector.prototype.getMeta = function() {
-	var  val = "";
-
-	var elem = document.getElementById( "hidden_wpseo_desc" );
-	if ( elem !== null ) {
-		val = elem.value;
-	}
-
-	return val;
-};
```

Comment:
`getMeta` is removed because it's exactly the same as `getSnippetMeta`. `getSnippetMeta` is kept because it is also used in packages/js/src/analysis/snippetEditor.js

Reply:
For the reviewer, please also double check if the removed method is not used anywhere else

Reply:
I couldn't find any other place where the `getMeta()` method was used either. :)